assigment4/assignment4.py

```diff
@@ -0,0 +1,54 @@
+class Tile:
+ 
+    def __init__(self, row, col, g):
+        self.ROW = row
```

Comment:
Style-nit: all-capital attribute names stand out as strange but they are not wrong of course.